lib/components/option-menus/index.js

```diff
@@ -66,16 +66,17 @@ module.exports = () => {
   delegate(document.body, '.dqpl-options-menu [role="menuitem"]', 'keydown', (e) => {
     const which = e.which;
     const target = e.delegateTarget;
+    const dropdown = closest(target, '[role="menu"]');
+    const id = dropdown && dropdown.id;
+    const trigger = id && document.querySelector(`.dqpl-options-menu-trigger[aria-controls="${id}"]`);
+    const menuOpen = trigger.getAttribute('aria-expanded') === 'true';
```

Comment:
sorry, I didn't see this on my first pass through the code but looks like we'll need to check if trigger exists here first:
```js
const menuOpen = trigger && (trigger.getAttribute('aria-expanded') === 'true')
```